package.json

```diff
@@ -3,6 +3,9 @@
   "version": "1.3.5",
   "description": "AST utility module for statically analyzing JSX",
   "main": "lib/index.js",
+  "files": [
+    "lib"
```

Comment:
OK, but I'd really like to understand why you think "the package is too large" is worse than "the package code is broken".